### PR TITLE
lookup: add koa

### PR DESCRIPTION
**Pull request by joyeecheung**
Popular web framework, widely used in production (at least in China AFAIK).

The master branch is for v1.x, which is based on generators and can be affected by V8 updates. v2.x  is based on async/await so can not be directly run by Node at the moment. This PR adds tests for v1.x.

### CODE REVIEW

File: lib/lookup.json [hunk 1]
```diff
@@ -343,5 +343,9 @@
   "microtime": {
     "replace": true,
     "prefix": "v"
+  },
+  "koa": {
```
Comment (TheAlphaNerd) [inline lib/lookup.json]: this needs node > 4.x right?. Can you add

"skip": "<4"

Comment (joyeecheung) [inline lib/lookup.json]: Sorry for missing that. skip field added.

File: lib/lookup.json [hunk 2]
```diff
@@ -343,5 +343,10 @@
...
+    "replace": true,
+    "master": true,
```
Comment (TheAlphaNerd) [inline lib/lookup.json]: We can remove this. `npm install` currently grabs v1.2.4

**Comment by joyeecheung** [comment]
Removed the master field. PTAL.

**Comment by gibfahn** [comment]
Ran CI on your branch (on all the modules) and koa passed everywhere: https://ci.nodejs.org/view/Node.js-citgm/job/citgm-smoker/484/

**Comment by TheAlphaNerd** [comment]
landed in 7f9dc0413d

**Pull request closed without merging**